lib/elixir/test/elixir/fixtures/dialyzer/protocol_opaque.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Dialyzer.ProtocolOpaque do
+  def circus do
```

Comment:
It isn't really but I believe @lexmag just wanted to make it consistent with the `def new()` below. Anyway, this looks good to go for me now. :)